src/js/utils/specs/date-utils-spec.js

```diff
@@ -0,0 +1,163 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../date-utils.js',
+        'moment',
+        'momenttimezone'
+    ],
+    function(
+        SpecHelpers,
+        DateUtils,
+        MomentLib,
+        MomentTZ
+    ) {
+        'use strict';
+
+        describe('DateUtils', function() {
+            beforeEach(function() {
+                setFixtures('<div class="test"></div>');
+            });
+
+            describe('DateObject', function() {
+                it('returns an empty string if provided no time', function() {
+                    expect(DateUtils.DateObject()).toEqual('');
+                });
+                it('convert a string to an object', function() {
+                    expect(DateUtils.DateObject('2016-10-14 08:00:00+00:00').format('ll')).toEqual('Oct 14, 2016');
+                });
+            });
+
+            describe('UserTimeZone', function() {
+                it('attempt to determine users timezone', function() {
+                    /**
+                     * we need to determine the system tz offset from which to compare
+                     *
+                     */
+                    var d = new Date();
+                    var e = d.getTimezoneOffset();
+                    var f = MomentTZ.tz.zone(DateUtils.UserTimeZone()).offset(d);
+                    expect(e).toEqual(f);
+                });
+                it('will take timezone input', function() {
+                    expect(DateUtils.UserTimeZone('America/New_York')).toEqual('America/New_York');
+                });
+            });
+
+            describe('LocalizedTime', function() {
+                it('test error conditions', function() {
+                    var t = DateUtils.DateObject('2016-10-14 08:00:00+00:00');
+                    expect(
+                        DateUtils.LocalizedTime()).toEqual('');
```

Comment:
If this needs to be broken across multiple lines, it should be three lines like so:

```javascript`
expect(
   foo
).toEqual('')
```

But it looks like this would fit under the line length limit if it was all just one line.